src/artemis/parameters/constants.py

```diff
@@ -0,0 +1,34 @@
+from enum import Enum
+from typing import Union
+
+from artemis.devices.fast_grid_scan import GridScanParams
+from artemis.devices.rotation_scan import RotationScanParams
+
+SIM_BEAMLINE = "BL03S"
+SIM_INSERTION_PREFIX = "SR03S"
+ISPYB_PLAN_NAME = "ispyb_readings"
+SIM_ZOCALO_ENV = "devrmq"
+DEFAULT_EXPERIMENT_TYPE = "grid_scan"
+
+PARAMETER_VERSION = 0.1
+
+EXPERIMENT_NAMES = ["grid_scan", "rotation_scan"]
+EXPERIMENT_TYPE_LIST = [GridScanParams, RotationScanParams]
+EXPERIMENT_DICT = dict(zip(EXPERIMENT_NAMES, EXPERIMENT_TYPE_LIST))
+EXPERIMENT_TYPES = Union[GridScanParams, RotationScanParams]
```

Comment:
Should: I think we should have a dataclass or some other kind of structure to define these to avoid having to go on the assumption `EXPERIMENT_NAMES[0]` corresponds to `EXPERIMENT_TYPE_LIST[0]`

Reply:
fair, I just swapped the dict and the lists so they are defined as a pair